### ISSUE TITLE
Data structures - array

### ISSUE THREAD

**Issue by msboffl** [msboffl/data-structures-and-algorithms#5]
Write a blog at [msboffl](https://msboffl.com/series/algorithms) for data structure - arrays

- [ ] What is an array
- [ ] How to create an array
- [ ] How to insert elements into an array
- [ ] How to delete elements into an array
- [ ] Add Problem links

**Comment by Pragya82001** [comment]
Hey Please assign me this task.

**Comment by msboffl** [comment]
Yeah sure, I will assign it to you, Can you please ping me on LinkedIn once, I will discuss some points with you on DSA & Credentials. 

I have started another project also [visualizer](https://github.com/msboffl/algorithm-visualizer)

**Comment by Pragya82001** [comment]
Okay. I will msg you on LinkedIn.

**Comment by msboffl** [comment]
Ok, Thank you @Pragya82001

**Comment by ishita126jain** [comment]
Hey @msboffl can I some advance array ques in this issue

**Comment by msboffl** [comment]
Hi, @Dishu099 I'm assigned this to you can work

**Comment by Dishu099** [comment]
Thanks

**Comment by Dishu099** [comment]
@msboffl  I have a doubt that Blog should be added in a Readme file of array folder ???
